resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesSuite.scala

```diff
@@ -152,6 +158,8 @@ private[spark] class KubernetesSuite extends SparkFunSuite with BeforeAndAfter {
       .set("spark.executor.cores", "1")
       .set("spark.executors.instances", "1")
       .set("spark.app.id", "spark-pi")
+      .set("spark.ui.enabled", "true")
```

Comment:
I think other tests need this too - wherever we invoke `expectationsForStaticAllocation()`. Although, part of improving the integration tests would be to find something other than the UI to rely on for checking the correctness.